### ISSUE TITLE
[FEATURE] Using variables for generated folder and file names

### ISSUE THREAD

**Issue by Moongazer** [CPS-IT/project-builder#150]
### Is your feature request related to a problem?

While generating project skeletons using Project-Builder, it would be very helpful if vendor-specific folders could be created. E.g. to include a base site-extension for the vendor in TYPO3 projects: `packages/myvendorname-site/`

### Describe the solution you'd like

A solution we could think of is the definition of a `source` and a `target` paths inside the `config.yaml`, e.g.:
```
filesystem:
  - source: 'packages/vendor-site/'
    target: 'packages/{{ project.vendor|slugify }}-site/'
```
This way would avoid placeholder-definitions in folder or file names checked into the template-repository which could lead to problems because of special-chars.

**Comment by eliashaeussler** [comment]
Hi @Moongazer, thanks again for this feature request! We've implemented this feature with #170 and released it as `2.2.0`.

You can now add the following file condition to your template configuration:

```diff
 steps:
   # ...
   - type: processSourceFiles
     options:
       fileConditions:
+        - path: 'packages/vendor-site/*'
+          target: 'packages/{{ project.vendor|slugify }}-site/*'
         # ...
 # ...
```

**Comment by Moongazer** [comment]
Amazing! Thanks a lot, will try it asap 👍🏼
